src/main/java/com/example/loosecoupling/controller/StudentController.java

```diff
@@ -0,0 +1,61 @@
+package com.example.loosecoupling.controller;
+
+
+import com.example.loosecoupling.dto.StudentCreationDto;
+import com.example.loosecoupling.dto.StudentDto;
+import com.example.loosecoupling.exception.InvalidPersonTypeException;
+import com.example.loosecoupling.mapstruct.StudentMapper;
+import com.example.loosecoupling.repository.StudentRepository;
+import com.example.loosecoupling.services.CollegeStudentsServices;
+import com.example.loosecoupling.services.ParentServices;
+import com.example.loosecoupling.services.ProfessorServices;
+import com.example.loosecoupling.services.StudentServices;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/college")
+public class StudentController {
+    //automatically dependancy injection create object for the StudentSerives
+    @Autowired
+    private StudentServices studentServices;
```

Comment:
Since there is only one StudentService please name the class and variable as StudentService/studentService
Also, since we already have college student service, do we need this?

Reply:
as college student service is interface so need such class which is going to declare all methods in interface,
I will changing name of college student service to college service.
An in StudentService provide two services like creating student, get all students information